### PR TITLE
saving licenses-data into sca-image-scan's report

### PR DESCRIPTION
**Pull request by itai1357**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
* saving licenses-data into sca-image-scan's report
* licenses name will be appear in the BOM-report
* violated-licenses will be appeared in the cli-output 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/sca_image/runner.py [hunk 1]
```diff
@@ -246,6 +248,31 @@ def get_image_report(self, dockerfile_path: str, image: Image, runner_filter: Ru
 
         return report
 
+    def get_license_statuses(self, packages: list[dict[str, Any]]) -> List[_LicenseStatus]:
+        requests_input = [
+            {"name": package.get("name", ""), "version": package.get("version", ""), "lang": package.get("type", "")}
+            for package in packages
+        ]
+        get_license_violation_url = f"{bc_integration.api_url}/api/v1/vulnerabilities/packages/get-licenses-violations"
```
Comment (Saarett) [inline checkov/sca_image/runner.py:256]: Can we put it as a class parameter instead?

File: tests/sca_image/test_runner.py [hunk 2]
```diff
@@ -65,3 +66,45 @@ def test_runner_honors_enforcement_rules(mock_bc_integration, image_name, cached
     assert summary["failed"] == 0
     assert summary["skipped"] == 3
     assert summary["parsing_errors"] == 0
+
+
+@responses.activate
+def test_licenses_status(mock_bc_integration):
```
Comment (Saarett) [inline tests/sca_image/test_runner.py:72]: Great test! WDYT about creating a test for a case the request fails?

**Review by Saarett — COMMENTED** [review]
Great job! Added some comments

File: tests/sca_image/test_runner.py (hunk 2 above)
Comment (itai1357) [reply tests/sca_image/test_runner.py:72]: sure!

File: checkov/sca_image/runner.py (hunk 1 above)
Comment (itai1357) [reply checkov/sca_image/runner.py:256]: sure! I will fix

**Review by Saarett — APPROVED** [review]

**Review by matansha — APPROVED** [review]

**Pull request merged**